Frends.Facebook.Request/Frends.Facebook.Request/Frends.Facebook.Request.cs

```diff
@@ -0,0 +1,159 @@
+﻿#pragma warning disable SA1200 //Using directives should be placed correctly.
+
+using Frends.Facebook.Request.Definitions;
+using Frends.HTTP.Request.Definitions;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Frends.Facebook.Request;
+
+/// <summary>
+/// Facebook class.
+/// </summary>
+public static class Facebook
+{
+    /// <summary>
+    /// This is task for reading data from Facebook API.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.Facebook.Get).
+    /// </summary>
+    /// <param name="input">Set reference type, parameters and token.</param>
+    /// <param name="options">Optional parameters.</param>
+    /// <param name="cancellationToken">Cancellation token given by Frends.</param>
+    /// <returns>Object { bool Success, dynamic Message }.</returns>
+    public static async Task<Result> Request([PropertyTab] Input input, [PropertyTab] Options options, CancellationToken cancellationToken)
+    {
+        if (string.IsNullOrEmpty(input.AccessToken))
+        {
+            throw new ArgumentNullException(nameof(input.AccessToken) + " cannot be empty.");
+        }
+        else if (string.IsNullOrEmpty(input.ApiVersion))
+        {
+            throw new ArgumentNullException(nameof(input.ApiVersion) + " cannot be empty.");
+        }
+        else if (string.IsNullOrEmpty(input.Reference))
+        {
+            throw new ArgumentNullException(nameof(input.Reference) + " cannot be empty.");
+        }
+        else if (string.IsNullOrEmpty(input.Message) && Enum.GetNames(typeof(SendMethods)).Contains(input.Method.ToString()))
+        {
+            throw new ArgumentNullException(nameof(input.Message) + " cannot be empty.");
+        }
+
+        var headers = GetHeaderDictionary(input);
+
+        using var content = GetContent(input, headers);
+        var url = $@"https://graph.facebook.com/v{input.ApiVersion}/" + (!string.IsNullOrEmpty(input.QueryParameters) ? input.Reference + "?" + input.QueryParameters : input.Reference);
+        using var responseMessage = await GetHttpRequestResponseAsync(
+                new HttpClient(),
+                input.Method.ToString(),
+                url,
+                content,
+                headers,
+                options,
+                cancellationToken)
+            .ConfigureAwait(false);
+
+        var hbody = string.Empty;
+
+#if NET471
+    hbody = responseMessage.Content != null ? await responseMessage.Content.ReadAsStringAsync().ConfigureAwait(false) : null;
+#elif NET6_0_OR_GREATER
+        hbody = responseMessage.Content != null ? await responseMessage.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false) : null;
+#endif
```

Comment:
What about netstandard2.0. It's not in this scope.

Reply:
Frends.Get was only targeted to framework and net 6.0, due to instructions

Reply:
This needs to be checked. I believe that with only net471 and net6 targets the Task can still only be run with cross-platform agents. The legacy agent needs also the netstandard2.0

Reply:
Nope, this works fine.